scalableminds/webknossos
# Uploads survive backend restarts

### URL of deployed dev instance (used for testing):
- https://robusteruploads.webknossos.xyz

### Steps to test:
- upload a dataset from the dashboard (try with single zip file, and with unpacked multi-file dataset)
- kill backend while upload is going
- restart backend
- => upload should continue and finish correctly 

### Follow-up
created follow up issue  #5849 for orphan cleanup

### Issues:
- fixes #5150 
- fixes #5056 

------
